opentrons-ai-client/src/components/organisms/Settings/Privacy.tsx

```diff
@@ -0,0 +1,57 @@
+import { useTranslation } from 'react-i18next'
+
+import {
+  ALIGN_CENTER,
+  COLORS,
+  DIRECTION_COLUMN,
+  Flex,
+  JUSTIFY_SPACE_BETWEEN,
+  ListItem,
+  SPACING,
+  StyledText,
+} from '@opentrons/components'
+
+import { ToggleButton } from '../../../molecules/ToggleButton'
+
+interface PrivacyProps {
+  enableAnalytics: boolean
+  onToggleAnalytics: () => void
+}
+
+export function Privacy({
+  enableAnalytics,
+  onToggleAnalytics,
+}: PrivacyProps): JSX.Element {
+  const { t } = useTranslation('protocol_generator')
+
+  return (
+    <Flex
+      flexDirection={DIRECTION_COLUMN}
+      gridGap={SPACING.spacing8}
+      height="100%"
```

Comment:
```suggestion
      height="100%"
```

need remove height 100% to align with PD's settings layout

![Screenshot 2025-06-27 at 10 42 38 AM](https://github.com/user-attachments/assets/140e6dd6-3d6d-435d-ab64-1ea5a3ae616d)

Reply:
Removed